Sources/Server/Server.swift

```diff
@@ -0,0 +1,42 @@
+import Foundation
+import Socket
+
+public class Server {
+    
+    private let port: Int
+    private let acceptNewConnection = true
+    private let requestBody: [String: String] = ["GET": "<!DOCTYPE html><html><body><h1>Hello World</h1></body></html>", "HEAD": ""]
+
+    public init(port: Int){
+        self.port = port
+    }
+
+    public func startServer() throws{
+        let socket = try Socket.create()
+        try socket.listen(on: port)
+        repeat {
+            let connectedSocket = try socket.acceptClientConnection()
+            let request = try connectedSocket.readString() 
+            let httpRequest = parseRequest(request: request!)
+            try connectedSocket.write(from: httpRequest.request)
+            connectedSocket.close()
+        } while acceptNewConnection
+    }
+
+    private func parseRequest(request: String) -> Request{
+        let requestMethod = request.components(separatedBy: " /")
+        return  Request(body: requestBody[requestMethod[0]]!)
+    }
+
+}
+
+struct Request {
```

Comment:
`Request` seems like it's being used also as a response. Let's look together later.